collect_app/src/test/java/org/odk/collect/android/utilities/TextUtilsTest.java

```diff
@@ -55,12 +72,12 @@ public void markDownToHtml_SupportsHtml() {
     @Test
     public void textToHtml_SupportsEscapedLt() {
         String[] tests = {
-                "<1",
+                ":gm; 1",
```

Comment:
Can you please explain this change and the related changes to the code? :gm; is not a valid character. The desired output is indeed &lt;

Reply:
That was suppose to be  "&gm; 1" not ":gm; 1" . The reason why I made those changes was because I changed the  text = text.replaceAll("<([^a-zA-Z/])", "&lt;$1"); from that to this  text = text.replaceAll("<([^a-zA-Z/])", "&gm;$1"); and because of that I also had to change all the &lt; to &gm; to make the tests run successfully.

After studying the codes again I realized that this method public void textToHtml_SupportsEscapedLt() in the TextUtilsTest.java class is for escaping lt; and not gm;
I have modified the codes now. Please review! @lognaturel